tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -484,6 +484,17 @@ batch:
         variables:
           AWS_LC_CI_TARGET: "tests/ci/run_formal_verification_saw_x86_64.sh"
 
+    # SAW proof for AES-GCM on platform x86
+    - identifier: ubuntu2004_clang10x_formal_verification_saw_x86_64_aes_gcm
+      buildspec: ./tests/ci/codebuild/common/run_simple_target.yml
```

Comment:
It'll be worth creating a new yml file that's capable of taking the extra argument for `./tests/ci/run_formal_verification.sh`. That way we could abstract out all the `tests/ci/run_formal_verification_*.sh` scripts and wouldn't need to maintain additional scripts.

The ec2-test-framework yml file [here](https://github.com/aws/aws-lc/blob/e89e5693171ecc484cca9af24e39914daff88f6c/tests/ci/codebuild/common/run_ec2_target.yml) might be a good example.

Reply:
That's a great idea! Check the most recent commit.